fluence/tests/compilation_tests/import_functions/ref_arrays.rs

```diff
@@ -42,6 +42,8 @@ extern "C" {
 
     pub fn string_type(arg: &Vec<String>) -> Vec<String>;
 
+    pub fn bool_type(arg: &Vec<bool>) -> Vec<bool>;
```

Comment:
i may be imagining things but i was under the impression that Vec<bool> is a no go.

Reply:
What do you mean? Please note that this's a test.